modules/cli/src/integrationTest/java/org/apache/ignite/internal/rest/ItGeneratedRestClientTest.java

```diff
@@ -90,7 +91,7 @@
 @ExtendWith(WorkDirectoryExtension.class)
 @MicronautTest(rebuildContext = true)
 @TestInstance(Lifecycle.PER_CLASS)
-public class ItGeneratedRestClientTest {
+public class ItGeneratedRestClientTest extends BaseIgniteAbstractTest {
```

Comment:
change required because of the import CliIntegrationTest.ALL_METRIC_SOURCES and Depends on Mockito check